### PR TITLE
[8.0][FIX] business_requirement_deliverable_project: Fix issue could not g…

### PR DESCRIPTION
**Pull request by martintorresvm**
…enerate projects.

Also fix the generated project is not linked anymore for deliverable lines.
	modified:   business_requirement_deliverable_project/models/business.py
	modified:   business_requirement_deliverable_project/models/project.py
	modified:   business_requirement_deliverable_project/wizard/generate_projects.py

### CODE REVIEW

**Comment by martintorresvm** [comment]
@elicoidal @dreispt

File: business_requirement_deliverable_project/models/project.py [hunk 1]
```diff
@@ -26,11 +26,16 @@ def generate_project_wizard(self):
...
+                    'approved',
+                    'stakeholder_approval',
+                    'cancel',
+                    'done'
+            ]:
                 raise ValidationError(
                     _("""All business requirements of the project should
                         be approved/canceled/done"""))
-            if br.state != 'approved':
+            if br.state not in ['approved', 'stakeholder_approval']:
```
Comment (elicoidal) [inline business_requirement_deliverable_project/models/project.py]: Shouldnot it  only be "Stakeholder approval"?
In principle it should be possible to generate project when only internally approved but this is dangerous somehow

**Review by elicoidal — CHANGES_REQUESTED** [review]

**Comment by martintorresvm** [comment]
@elicoidal @rafaelbn @gurneyalex @dreispt Please review.

**Comment by gurneyalex** [comment]
please fix the tests.

**Comment by dreispt** [comment]
@victormartinelicocorp tests are failing.

**Comment by elicoidal** [comment]
@victormartinelicocorp @sudhir-serpentcs Can you check the travis tests so that we can merge?

**Comment by martintorresvm** [comment]
> @victormartinelicocorp @sudhir-serpentcs Can you check the travis tests so that we can merge?

Done @elicoidal

File: business_requirement_deliverable_project/models/project.py [hunk 2]
```diff
@@ -26,11 +26,15 @@ def generate_project_wizard(self):
...
         for br in br_ids:
-            if br.state not in ['approved', 'cancel', 'done']:
+            if br.state not in [
+                    'stakeholder_approval',
+                    'cancel',
+                    'done'
+            ]:
                 raise ValidationError(
                     _("""All business requirements of the project should
                         be approved/canceled/done"""))
```
Comment (elicoidal) [inline business_requirement_deliverable_project/models/project.py]: Approved by Stakeholder

**Review by elicoidal — APPROVED** [review]
Slight modification

**Comment by martintorresvm** [comment]
> Slight modification

@elicoidal

**Comment by elicoidal** [comment]
@dreispt  all green here: can we merge?

**Review by astirpe — APPROVED** [review]

**Pull request merged**